projects/js-packages/connection/components/connect-screen/required-plan/style.scss

```diff
@@ -56,7 +56,7 @@
 			text-decoration: underline;
 
 			width: auto;
-			min-width: 0;
+			min-width: fit-content;
```

Comment:
Hi @jwebbdev, thanks for working on this!

`fit-content` doesn't work in Firefox.
We could use `-moz-fit-content`, but that would require the whole new level of CSS trickery, so I wouldn't recommend that either.
Is there a way to solve this without using `fit-content`?

<img width="552" alt="Screen Shot 2022-01-10 at 1 39 09 PM" src="https://user-images.githubusercontent.com/1341249/148820676-7b33fd3b-a902-4d72-beb3-acdffef0d458.png">

Reply:
I can switch it back to 0.  I updated that based on another suggestion but I believe 0 worked fine as well.